sound/soc/sof/ipc4-pcm.c

```diff
@@ -600,6 +600,11 @@ static void sof_ipc4_build_time_info(struct snd_sof_dev *sdev, struct snd_sof_pc
 		}
 	}
 
+	if (!host_copier || !dai_copier) {
+		dev_err(sdev->dev, "host or dai copier are not found");
```

Comment:
or both missing...
It could help to say which one is missing?

In this function only the dai_copier is needed, so perhaps a comment to say that for the time_info to work both host and dai copier must be valid?

Reply:
it is also a good to have `\n` at the end of the string.

Reply:
good idea